burn-book/src/SUMMARY.md

```diff
@@ -13,16 +13,15 @@
   - [Tensor](./building-blocks/tensor.md)
   - [Autodiff](./building-blocks/autodiff.md)
   - [Module](./building-blocks/module.md)
-  - [Config]()
-  - [Learner]()
-  - [Metric]()
+  - [Learner](./building-blocks/learner.md)
+  - [Metric](./building-blocks/metric.md)
   - [Record]()
   - [Dataset]()
 - [Import ONNX Model]()
 - [Advanced](./advanced/README.md)
   - [Backend Extension](./advanced/backend-extension/README.md)
     - [Custom WGPU Kernel](./advanced/backend-extension/custom-wgpu-kernel.md)
-  - [Custom Training Loops]()
+  - [Custom Training Loops](./advanced/custom-training-loop.md)
```

Comment:
Unless we plan to complete before the release we should keep the section empty, so that the user knows not to click on the link.

Reply:
I plan to complete it :)